VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -60,14 +86,89 @@ def get_loop_devices(self):
                 disk_loop_devices_file_systems.append(file_system_info[0])
         return disk_loop_devices_file_systems
   
+    def disk_list_for_billing(self):
+        if(len(self.lsscsi_list) != 0):
+            for item in self.lsscsi_list:
+                idxOfColon = item.rindex(':',0,item.index(']'))# to get the index of last ':'
+                idxOfColon += 1
+                lunNumber = int(item[idxOfColon:item.index(']')])
+                # item_split is the list of elements present in the one row of the cmd sudo lsscsi
+                self.item_split = item.split()
+                #storing the corresponding device name from the list
+                self.device_name = self.item_split[len(self.item_split)-1]
+
+                for device in self.root_devices :
+                    if self.device_name in device :
+                        lunNumber = -1
+                        # Changing the Lun# of OS Disk to -1
+
+                if lunNumber in self.includedLunList :
+                    self.disksToBeIncluded.append(self.device_name)
+                self.logger.log("LUN Number {0}, disk {1}".format(lunNumber,self.device_name))   
+            self.logger.log("Disks to be included {0}".format(self.disksToBeIncluded))
```

Comment:
@arisettisanjana can you please share a sample extension trace file with these changes, will make it easier to correlate the command outputs and review

Reply:
[Extension_PR_log.txt](https://github.com/Azure/azure-linux-extensions/files/9763942/Extension_PR_log.txt)